src/GameWindow.py

```diff
@@ -0,0 +1,90 @@
+# This Python file uses the following encoding: utf-8
+import logging
+import sys
+
+from PyQt5 import QtWidgets
+from PyQt5.QtWidgets import QGraphicsBlurEffect
+
+from gamewindow_ui import Ui_GameWindow
+
+
+class GameWindow(QtWidgets.QMainWindow):
```

Comment:
Переделать в QWidget!

```suggestion
class GameWindow(QtWidgets.QWidget):
```

Reply:
:(
Ну лан((